src/commands/watch.ts

```diff
@@ -49,20 +50,28 @@ async function watch(configurations: string[]) {
 
           let confs = specMap[p];
           if (confs) {
-            confs.forEach(async (conf) => {
-              const outputs = await processConfiguration(conf);
-              outputs.forEach((output) => writeOutput(output));
-            });
+            for (const conf of confs) {
+              try {
+                const outputs = await processConfiguration(conf);
+                outputs.forEach((output) => writeOutput(output));
```

Comment:
Missed this before accepting, but `write output` is async as well. You can expand the loop like you did with the outer forEach or you can write it like `await Promise.all(outputs.map(writeOutput));`